### PR TITLE
:bug: Fix enter on empty search page

### PR DESCRIPTION
**Pull request by pabloalba**
Related to https://tree.taiga.io/project/penpot/issue/2597

### CODE REVIEW

**Comment by niwinz** [comment]
I think the `[DASHBOARD]` is unnecessary on commit message. And the commit should be formatted like `Fix enter on empty search page` or something similar

File: frontend/src/app/main/ui/dashboard/sidebar.cljs [hunk 1]
```diff
@@ -169,7 +170,13 @@
            (let [search-input (dom/get-element "search-input")]
              (dom/clean-value! search-input)
              (dom/focus! search-input)
-             (emit! (dd/go-to-search)))))]
+             (emit! (dd/go-to-search)))))
+        on-key-press
```
Comment (niwinz) [inline frontend/src/app/main/ui/dashboard/sidebar.cljs:175]: Missing white space separation :lipstick:

Comment (pabloalba) [reply frontend/src/app/main/ui/dashboard/sidebar.cljs:175]: fixed

**Comment by pabloalba** [comment]
> I think the `[DASHBOARD]` is unnecessary on commit message. And the commit should be formatted like `Fix enter on empty search page` or something similar

Fixed

**Pull request merged**